OutOfSchool/OutOfSchool.AuthCommon/Services/IdentityRolesInitializerHostedService.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Microsoft.Extensions.Hosting;
+
+namespace OutOfSchool.AuthCommon.Services;
+
+/// <summary>
+/// Hosted service that initializes default roles for correct work of authorization server.
+/// </summary>
+public class IdentityRolesInitializerHostedService : IHostedService
+{
+    private readonly IServiceProvider serviceProvider;
+    private readonly ILogger<IdentityRolesInitializerHostedService> logger;
+
+    public IdentityRolesInitializerHostedService(
+        IServiceProvider serviceProvider,
+        ILogger<IdentityRolesInitializerHostedService> logger)
+    {
+        this.serviceProvider = serviceProvider;
+        this.logger = logger;
+    }
+
+    /// <summary>
+    /// Get default role names that should be initialized for correct work of authorization server.
+    /// </summary>
+    /// <returns>New array that contains default role names.</returns>
+    public static string[] GetDefaultRoleNames() =>
+    [
+        "parent",
+        "provider",
+        "techadmin",
+        "ministryadmin",
+        "regionadmin",
+        "areaadmin",
+    ];
+
+    public async Task StartAsync(CancellationToken cancellationToken)
+    {
+        using var scope = serviceProvider.CreateScope();
+
+        string[] defaultRoleNames = GetDefaultRoleNames();
```

Comment:
(Code style) Please use for local variables everywhere either `var` or explicit types, not both (I have reasons to use `var`, but don't insist). Current approach is bit inconsistent and noisy.